apps/antalmanac/src/routes/Error.tsx

```diff
@@ -0,0 +1,25 @@
+import { Typography, Button, Stack, Box } from '@mui/material';
+import { Link } from 'react-router-dom';
+
+const ErrorPage = () => (
+    <Box sx={{ padding: '1rem' }}>
+        <Typography variant="h3" component="h1">
+            Oops! Something went wrong 🥺 👉👈
```

Comment:
```suggestion
            Oops! Something went wrong.
```
nit: 🥺 as much as I'd like to keep the emojis, I think AA broadly tries to maintain a professional tone / communication